src/WingetCreateCLI/Program.cs

```diff
@@ -110,12 +110,16 @@ private static void DisplayHelp(NotParsed<object> result)
 
         private static void DisplayParsingErrors<T>(ParserResult<T> result)
         {
-            var builder = SentenceBuilder.Create();
-            var errorMessages = HelpText.RenderParsingErrorsTextAsLines(result, builder.FormatError, builder.FormatMutuallyExclusiveSetErrors, 1);
-
-            foreach (var error in errorMessages)
+            NotParsed<object> notParsed = result as NotParsed<object>;

```

Comment:
Why don't you just change the function parameter to be NotParsed<object>

Reply:
Changed function parameter to NotParsed

Reply:
Changed function parameter to NotParsed